### PR TITLE
[enterprise-4.15] OSDOCS#9744: Update release notes for Hosted control planes

### PR DESCRIPTION
**Pull request by xenolinux**
Now that the ACM 2.10 is GA, this PR updates the hosted control planes release notes

Version(s): 4.15 

Issue: https://issues.redhat.com/browse/OSDOCS-9744

Link to docs preview: 
- Hosted Control Planes new features: https://73867--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-hcp

- Known issues: https://73867--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-known-issues

QE review:
- [x] QE has approved this change.

Additional information:
 This PR is a part of CM

### CODE REVIEW

**Review by jparrill — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2414,13 +2429,18 @@ In the following tables, features are marked with the following statuses:
...
+|General Availability
 |General Availability
 // Needs to move to GA after Nov 15, 2023
 |Hosted control planes for {product-title} on {VirtProductName}
 |Not Available
-|Technology Preview
 |General Availability
+|General Availability
+
+|Hosted control planes for {product-title} using the non bare metal agent machines
```
Comment (zanetworker) [inline release_notes/ocp-4-15-release-notes.adoc:2440]:
```suggestion
|Hosted control planes for {product-title} using non bare metal agent machines
```

**Comment by davemulford** [comment]
LGTM

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (xenolinux) [reply release_notes/ocp-4-15-release-notes.adoc:2440]: Fixed

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1018,9 +1018,24 @@ For more information, see xref:../scalability_and_performance/cnf-performing-pla
...
+
+[id="ocp-4-15-non-bare-metal-agent-machines"]
+==== Configuring hosted control plane clusters using non bare metal agent machines (Technology Preview)
+
+With this release, you can provision a hosted control plane cluster by using non bare metal agent machines. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-agent-non-bm[Configuring hosted control plane clusters using non bare metal agent machines (Technology Preview)].
+
+[id="ocp-4-15-hcp-kubevirt-console"]
+==== Creating a hosted cluster by using the console
+
+With this release, you can now create a hosted cluster with the KubeVirt platform by using the console. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosted-create-kubevirt-console[Creating a hosted cluster by using the console].
```
Comment (zanetworker) [inline release_notes/ocp-4-15-release-notes.adoc:1033]:
```suggestion
With this release, you can now create a hosted cluster with the KubeVirt platform by using the Advanced Cluster Manager console, also available with the Multicluster Engine for Kubernetes Operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosted-create-kubevirt-console[Creating a hosted cluster by using the console].
```

**Comment by zanetworker** [comment]
tiny nit, otherwise LGTM

**Review by adellape — COMMENTED** [review]
Added a few peer-review-style comments, but +1 for the CM ack (on DPM behalf as Kathryn is currently OoO).

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -2414,13 +2429,18 @@ In the following tables, features are marked with the following statuses:
...
+|General Availability
 |General Availability
 // Needs to move to GA after Nov 15, 2023
 |Hosted control planes for {product-title} on {VirtProductName}
 |Not Available
-|Technology Preview
 |General Availability
+|General Availability
+
+|Hosted control planes for {product-title} using the non-bare metal agent machines
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:2440]: Looks like this earlier fix might have been reverted?
```suggestion
|Hosted control planes for {product-title} using non-bare metal agent machines
```

**Review by adellape — APPROVED** [review]

**Pull request merged**